_posts/2016-06-20-sustaining-hoodie.md

```diff
@@ -0,0 +1,91 @@
+---
+
+title: 'We are here for the long run: growing a sustainable Hoodie community'
+author: Gregor
+layout: blog-post
+comments: true
+
+---
+
+When we started working on Hoodie back in 2012, we understood its potential. The technology was promising, and it still excites us, to this day. Since the beginning we have been thinking a lot about how to grow Hoodie beyond ourselves, and we might have found the right tool to help us get there, together.
+
+OpenCollective allows the Hoodie community to collect money from backers and sponsors in a transparent way, without the need to create a legal entity. This is huge, because we can focus on finding backers and sponsors, instead of using our resources for legal matters.
+
+**Join us on our mission to empower everyone to express themselves with technology**, become a backer (starting at just $1 per month!) or an official sponsor (starting at $100 per month).
+
+We have a lot more to say about this topic, please read on :)
+
+## Experience
+
+[Alex](https://twitter.com/espylaub), [Jan](https://twitter.com/janl) and [myself](https://twitter.com/gr2m) worked on other successful Open Source projects before starting Hoodie in 2012. We have seen projects thrive, and we have seen them fade away. We have seen people spending a lot of time moving a project forward, and we’ve seen people harassed and friends burned out.
+
+Based on our experience, we run things differently with Hoodie. We know that technology is not the biggest challenge. Maintaining it, creating an inclusive community around it, and growing and sustaining it, while keeping it independent, are the biggest challenges.
+
+We think that a high adoption rate should not be Hoodie’s primary goal. If an independent Open Source project becomes too successful too fast, [the balance between users, contributors, and maintainers becomes unhealthy](https://medium.com/the-javascript-collection/healthy-open-source-967fa8be7951). A recipe for individual burnout and project failure.
```

Comment:
changed it to 

> If an independent Open Source project becomes too successful too fast, [the balance between users, contributors, and maintainers becomes unhealthy](https://medium.com/the-javascript-collection/healthy-open-source-967fa8be7951); it’s a recipe for individual burnout and project failure.

in the google docs draft: https://docs.google.com/document/d/1GTsJwZti9P3XaoOJ2o-qXTZ0rVUHiiB0CAe49bCKkPo/edit